src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.Portable.cs

```diff
@@ -34,10 +34,14 @@ public static partial class ThreadPool
         // Threadpool specific initialization of a new thread. Used by OS-provided threadpools. No-op for portable threadpool.
         internal static void InitializeForThreadPoolThread() { }
 
+#pragma warning disable IDE0060
         internal static bool CanSetMinIOCompletionThreads(int ioCompletionThreads) => true;
-        internal static void SetMinIOCompletionThreads(int ioCompletionThreads) { }
-
         internal static bool CanSetMaxIOCompletionThreads(int ioCompletionThreads) => true;
+#pragma warning restore IDE0060
+
+        [Conditional("unnecessary")]
```

Comment:
Some of the similar cases are partial methods. Is there are reason why these are `Conditional("unnecessary")` and not partial methods?

Reply:
They are not private hence we cannot use a partial method without an implementation.